admin/RunIncrementalJSONDump

```diff
@@ -23,15 +23,6 @@ source admin/config.sh
 . ./admin/functions.sh
 make_temp_dir
 
-# The database from which the dumps are generated is replicated, but created
-# as standalone - meaning it has foreign keys and triggers. Foreign keys are
-# needed for MusicBrainz::Server::Role::FollowForeignKeys to work. The
-# triggers aren't needed and could (or should) be dropped, but it's safer to
-# make sure they are fully disabled here too. Leaving them enabled would
-# certainly cause the data to become corrupted and replication to break.
```

Comment:
Do we need to disable *some* triggers so that they don't "cause the data to become corrupted and replication to break" or was this an unfounded fear all along?

Reply:
Well, I actually removed all triggers (except for the mirror-only ones) from the `musicbrainz_json_dump` DB, so there is nothing to disable anymore. :) And yeah, any trigger that modified replicated tables would cause replication to break.